experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -945,6 +945,11 @@ def validate(self, data):
                 }
             )
 
+        if self.should_call_preview_task and not self.instance.can_publish_to_preview:
+            raise serializers.ValidationError(
+                {"status": "This experiment cannot be published to preview."}
```

Comment:
Oh I didn't realize we have all these error messages just hanging out in the code here, these should probably be over in constants.  No need to worry about it in this pr tho.